robyn/types.py

```diff
@@ -18,5 +20,34 @@ def as_list(self):
         ]
 
 
+PathParams = NewType("PathParams", Dict[str, str])
+RequestMethod = NewType("RequestMethod", str)
+RequestURL = NewType("RequestURL", Url)
+FormData = NewType("FormData", Dict[str, str])
+RequestFiles = NewType("RequestFiles", Dict[str, bytes])
+RequestIP = NewType("RequestIP", Optional[str])
+RequestIdentity = NewType("RequestIdentity", Optional[Identity])
+
+
 class JSONResponse(TypedDict):
+    """
+    A base class to override to implement openapi response bodies
```

Comment:
Not really, these are not just base classes. These are actual classes, which can be used for type inference.